### PR TITLE
Exclude bots from leaderboard

### PR DESCRIPTION
**Pull request by KevinBacabac**
https://github.com/uwcsc/codeybot/issues/265

I don't have a database to test this ticket

### CODE REVIEW

**Review by Picowchew — COMMENTED** [review]
In terms of not having a database, shouldn't SQLite be installed due to being found in yarn.lock?

**Comment by Picowchew** [comment]
I might as well add that the change would be taking place in the messageCreate listener instead of the leaderboard, since bots shouldn't have a coin balance *at all*. Bots not appearing in the leaderboard would just be one effect of this.

**Comment by Picowchew** [comment]
The issue has been updated just now for clearer specifications; sorry about that!

**Comment by KevinBacabac** [comment]
No problem it's ready to test

File: src/listeners/messageCreate.ts [hunk 1]
```diff
@@ -112,6 +113,11 @@ export class MessageCreateListener extends Listener {
       return;
     }
```
Comment (Picowchew) [inline src/listeners/messageCreate.ts:114]: This case can be removed, since it is already covered below.

**Review by Picowchew — CHANGES_REQUESTED** [review]

**Comment by Picowchew** [comment]
Also, do you mind adding 275813801792634880 as IRC_USER_ID for staging and updating README.md for the new config variable IRC_USER_ID?

**Review by Picowchew — APPROVED** [review]
LGTM!

**Pull request merged**